src/transaction/utils/ccJsonLoad.js

```diff
@@ -22,22 +22,19 @@ export default function ccJsonLoad(conditionJson) {
         if (conditionJson.type === 'threshold-sha-256') {
             fulfillment = new cc.ThresholdSha256()
             fulfillment.threshold = conditionJson.threshold
-            conditionJson.subfulfillments.forEach((subfulfillmentJson) => {
-                const subfulfillment = ccJsonLoad(subfulfillmentJson)
-                if ('getConditionUri' in subfulfillment) {
-                    fulfillment.addSubfulfillment(subfulfillment)
-                } else if ('serializeUri' in subfulfillment) {
-                    fulfillment.addSubcondition(subfulfillment)
+            conditionJson.subconditions.forEach((subconditionJson) => {
+                const subcondition = ccJsonLoad(subconditionJson)
+                if ('getConditionUri' in subcondition) {
+                    fulfillment.addSubfulfillment(subcondition)
+                } else if ('serializeUri' in subcondition) {
+                    fulfillment.addSubcondition(subcondition)
                 }
             })
         }
 
         if (conditionJson.type === 'ed25519-sha-256') {
             fulfillment = new cc.Ed25519Sha256()
             fulfillment.publicKey = new Buffer(base58.decode(conditionJson.public_key))
-            if (conditionJson.signature) {
```

Comment:
Well, the `output.condition` doesn't seem like the right place to have a signature.